react/components/StorePickupItem.tsx

```diff
@@ -22,7 +22,7 @@ const StorePickupItem: FC<Props> = ({ store }) => {
     <div className={`flex flex-column t-body lh-copy c-muted-2 ${handles.pickupItem}`}>
       <span className={`t-heading-6 c-on-base ${handles.pickupName}`}>{`${friendlyName}`}</span>
       <span className={handles.pickupAddress}>{`${address.street}${address.number ? `, ${address.number}` : ''}`}</span>
-      <div className={`${shippingEstimate ? handles.pickupEstimate : handles.pickupUnavailable} flex`}>
+      <div className={`${shippingEstimate ? `${handles.pickupEstimate} ${handles.pickupEstimate}--${shippingEstimate}` : handles.pickupUnavailable} flex`}>
```

Comment:
You can use [applyModifiers](https://github.com/vtex-apps/stack-layout/blob/df8c378e93c2bd971209a7a271c8bc5f3eca0af8/react/index.tsx#L31)

```suggestion
<div className={`${shippingEstimate ? applyModifiers(handles.pickupEstimate, shippingEstimate) : handles.pickupUnavailable} flex`}>
```